### PR TITLE
[openui5] Update the definition files for OpenUI5 1.119

### PR DESCRIPTION
**Pull request by akudev**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).  → as detailed in the [previous pull request for 1.90](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53680), some are unavoidable at this point
- [x] [Run \`npm test <package to test>\`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: → no specific URL about the changes, it's just a new release: https://sdk.openui5.org/
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by petermuessig — APPROVED** [review]
The `showAsText` property introduced in 1.118 has been removed again? 

Rest looks good!

File: types/openui5/sap.tnt.d.ts [hunk 1]
```diff
@@ -50,20 +50,19 @@ declare module "sap/tnt/InfoLabel" {
...
-   * vertically aligned with UI5 Input and Button control families. When using `InfoLabel` in non-editable
-   * `Forms`, `Tables`, etc., set `displayOnly=true` for best visual results.
+   * The control visualizes text information without user interaction. The text-background color pair can
+   * be changed by setting a number between 1 and 10 that corresponds to the 10 predefined color combinations
+   * of the `colorScheme` property. The control is designed to be vertically aligned with UI5 Input and Button
+   * control families. When using `InfoLabel` in non-editable `Forms`, `Tables`, etc., set `displayOnly=true`
+   * for best visual results.
    *
    * Usage Guidelines:
    * 	 - If the text is longer than the width of the control, it doesn’t wrap. Instead, it’s represented as
```
Comment (DangerBotOSS) [inline types/openui5/sap.tnt.d.ts:60]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

File: types/openui5/sap.ui.mdc.d.ts [hunk 2]
```diff
@@ -1,4 +1,148 @@
...
+  import Control1 from "sap/ui/core/Control";
+
+  /**
+   * @since 1.82.0
+   *
+   * Base delegate implementation for {@link sap.ui.mdc.Control MDC Controls}. This delegate serves as base
+   * implementation for aggregation-based controls, such as:
+   *
+   *
+   * 	 - `sap.ui.mdc.Chart`
```
Comment (DangerBotOSS) [inline types/openui5/sap.ui.mdc.d.ts:15]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Review by petermuessig — APPROVED** [review]

**Comment by akudev** [comment]
Ready to merge

**Pull request merged**